ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -9,12 +9,14 @@ import Blocks from '#app/legacy/containers/Blocks';
 import Paragraph from '#app/legacy/containers/Paragraph';
 import UnorderedList from '#app/legacy/containers/BulletedList';
 import LivePageMediaPlayer from '#app/legacy/containers/LivePageMediaPlayer';
+import LegacyMediaPlayer from '#app/legacy/containers/LivePageMediaPlayer/legacyMediaPlayer';
```

Comment:
Might we consider renaming the parent component to just `MediaPlayer` instead of `LivePageMediaPlayer`?

Reply:
I think the idea we had is that all page types would create their own "wrapper" for the media player if its used in the page, so that custom styling can be more easily applied. This would allow the actual `MediaPlayer` component, which renders the actual player, to be completely generic and agnostic to the where it is placed.

We should probably move the `LivePageMediaPlayer` out of `legacy/containers` though to somewhere more appropriate.